src/gpu/sycl/sycl_primitive_conf.hpp

```diff
@@ -173,8 +173,7 @@ struct sycl_layer_normalization_conf_t {
     xpu::sycl::md_t scale;
     xpu::sycl::md_t shift;
     xpu::sycl::md_t stat_md;
-    xpu::sycl::md_t stat_d;
-    xpu::sycl::md_t var_md;
+    data_type_t var_dt;
```

Comment:
It seems that current implementations are just not sophisticated enough to utilize the rest of md knowledge which may (probably) be required once advanced blocking will be added on top.

I'm fine to replace md with dt, just be cautious that it might be needed back at some time in the future.

Reply:
Makes sense, thanks.